.github/workflows/mingw-w64-tiledb/PKGBUILD

```diff
@@ -48,5 +53,21 @@ package() {
 
 check (){
   cd ${source_dir}/build-${MINGW_CHOST}
-  #make check
+  # may want to add
+  #  -DTILEDB_VERBOSE=ON \
+  # to cmake options above if enabling/running 'make check'.
+  # install(/run) minio if invoking 'check' later since building with TILEDB_S3=ON
+  export MAGIC=${MINGW_PREFIX}/share/misc/magic.mgc
+  export TILEDB_VFS_S3_CONNECT_TIMEOUT_MS=6000
```

Comment:
Running tests against rtools/msys builds is great, but please move these changes to a separate PR where we can evaluate thoroughly.

Reply:
changes delayed to (pre-existing) PR SC15835